tests/cypress/support/commands.js

```diff
@@ -1833,3 +1846,30 @@ Cypress.Commands.add('applyActionToSliders', (wrapper, slidersClassNames, action
     });
     cy.get('.ant-tooltip').invoke('hide');
 });
+
+Cypress.Commands.add('mergeConsensusTask', (status = 202) => {
+    cy.intercept('POST', '/api/consensus/merges**').as('mergeJobs');
+
+    cy.get('.cvat-task-details-wrapper').should('be.visible');
+    cy.contains('button', 'Actions').click();
+    cy.contains('Merge consensus jobs').should('be.visible').click();
+    cy.get('.cvat-modal-confirm-consensus-merge-task')
+        .contains('button', 'Merge')
+        .click();
+
+    cy.wait('@mergeJobs').its('response.statusCode').should('eq', status);
+});
+
+Cypress.Commands.add('mergeConsensusJob', (status = 202) => {
+    cy.intercept('POST', '/api/consensus/merges**').as('mergeJob');
+
+    cy.get('.cvat-job-item').first().within(() => {
```

Comment:
It would be better to specify what job we want to merge, not just first one. Maybe ID parameter will work